src/request-type.ts

```diff
@@ -204,3 +204,11 @@ export namespace TextDocumentDidChangeNotification {
 		}
 	};
 }
+
+export namespace TextDocumentCompletionRequest {
+	export const type: vscode.RequestType<vscode.TextDocumentPositionParams, vscode.CompletionList, void> = {
+		get method() {
```

Comment:
Style inspired by `vscode-languageserver` which holds original request definitions. For example,
https://github.com/Microsoft/vscode-languageserver-node/blob/5732a65f313e77af6ef55611e9a63dc4311a47c8/client/src/protocol.ts#L79